persistence/db.go

```diff
@@ -26,50 +26,108 @@
 package persistence
 
 import (
-	"fmt"
-
 	"clouditor.io/clouditor/api/auth"
 	"clouditor.io/clouditor/api/orchestrator"
+	"fmt"
 	"github.com/sirupsen/logrus"
 	"gorm.io/driver/postgres"
 	"gorm.io/driver/sqlite"
 	"gorm.io/gorm"
 )
 
 var log *logrus.Entry
-var db *gorm.DB
+
+// TODO(lebogg): Lack for better term
+type GormX struct {
```

Comment:
This should probably be in a separate folder? so that we have a package `persistance/gorm` and `persistance/somethingelse`. At *least* it should be in a separate file.

Reply:
And the Name is weird :)